mmdet/apis/ote/apis/detection/ote_utils.py

```diff
@@ -71,10 +72,10 @@ def __len__(self):
 def generate_label_schema(label_names):
     label_domain = Domain.DETECTION
     colors = ColorPalette(len(label_names)) if len(label_names) > 0 else []
-    not_empty_labels = [LabelEntity(name=name, color=colors[i], domain=label_domain, id=i) for i, name in
+    not_empty_labels = [LabelEntity(name=name, color=colors[i], domain=label_domain, id=ID(str(i))) for i, name in
```

Comment:
Please, use `ID(i)` everywhere instead of `ID(str(it))`. It is simpler.

Reply:
thanks, simplified id parameter initialization